WookieeMatt/Essence20
# Jump Through Time Updates

##### In this PR
- Origins, Influences, and Perks from A Jump Through Time
- Mandatory Hang-Up option on Influences, even if it is your first Influence you still get the Hang-Up
- Added new Weapon Traits

##### Testing
- Run `fvtt package pack "jttitems"`
- Create Influence with a hang Up and Mandatory Hang-Up checked make sure that the Hang Up is added even if it is the first influence. 
- You can also test with the Beloved Influence in the Compendium. 
- Double check Compendium Items for any issues.
